src/components/dashboard/Dashboard.js

```diff
@@ -125,20 +123,25 @@ const FeedTab = ({ setActiveTab, getFeedPage, activeTab, tab }) => {
   )
 }
 
+const BridgeButton = () => (
+  <IconButton name="bridge" size={30} color="transparent" disabled={false} circle={false} customColor={'#00AEFF'} />
```

Comment:
try to use color from theme. look at the basic theme colors (we extend from)

Reply:
but I believe that transparent is already that no?
defaultTheme only contains hexes

so then I would now extend our theme with: rgba(0,0,0, 0) with a transparent name?

Reply:
@L03TJ3  was talking about 00AEFF

Reply:
done

Reply:
@johnsmith-gooddollar ye realized a minute later xD